docs/git/cloning.md

```diff
@@ -58,8 +58,8 @@ First, show the currently configured remote repository:
 
 ```
 $ git remote -v
-origin  git@github.com:YOUR_USERNAME/zulip.git (fetch)
-origin  git@github.com:YOUR_USERNAME/zulip.git (push)
+origin git@github.com:YOUR_USERNAME/zulip.git (fetch)
+origin git@github.com:YOUR_USERNAME/zulip.git (push)
```

Comment:
These changes are incorrect; Git has its own algorithm for how these are formatted, and it involves justifying to multiples of 8 spaces, at least on my system.